### PR TITLE
feature(#6): initial structure

### PR DESCRIPTION
**Pull request by FilipLeTran**

### CODE REVIEW

**Review by emilwihlander — CHANGES_REQUESTED** [review]
Issue #6 were initially created for html structure to have links to about ("Om sektionen"), for companies ("För företag"), etc. not to address actual lay-out. This pull request is therefore somewhat miss directed. It also isn't mobile-friendly.

**Comment by FilipLeTran** [comment]
Made the page responsive and added basic html structure. Also added CSS for the general layout and responsiveness.

File: src/index.html [hunk 1]
```diff
@@ -4,13 +4,38 @@
...
-    
-    <body class="pageHeader">
-        <div>
-            <h1>Welcome to dsek.se 2.0</h1>
-            100% php Free
-        </div>
+    <body>
+        
+        <nav class="navbar">
+            <img class="logo" src="pictures/D-symbol.svg" alt="D-sek">
```
Comment (frednordell) [inline src/index.html]: alt="D-sek symbol" ?

File: src/index.html [hunk 2]
```diff
@@ -4,13 +4,38 @@
...
+            </div>
+        </nav>    
+
+        <header class="main-header">
+            <div>D-sektionen inom TLTH</div>
+            <div>Datateknik && Information- och kommunikationsteknik</div>
+            <div>I want to go outside let me go outside nevermind inside is better really likes hummus, 
+                fight an alligator and win human clearly uses close to one life a night no one naps that 
+                long so i revive by standing on chestawaken!
+            </div>
```
Comment (frednordell) [inline src/index.html]: Should we standardize the lorem ipsum? @emilwihlander

**Review by emilwihlander — APPROVED** [review]
Looked at it now, this is a great start!

**Pull request merged**